src/Adapter/Driver/Oci8/Statement.php

```diff
@@ -34,6 +34,9 @@ class Statement implements StatementInterface, Profiler\ProfilerAwareInterface
 
     /** @var resource */
     protected $resource;
+    
+    /** @var bool */
+    protected $parametersBound;
```

Comment:
The newly introduced property should be marked `@internal` and `@deprecated`, to prevent downstream usage.

It should also be `public` for better compatibility with pre-existing codebases.

You will likely need to re-generate the psalm baseline, after this change.